packages/components/virtual-list/src/hooks/use-cache.ts

```diff
@@ -4,14 +4,20 @@ import memoOne from 'memoize-one'
 
 import type { VirtualizedProps } from '../props'
 
-export const useCache = () => {
+export const useCache = <T>() => {
   const vm = getCurrentInstance()!
 
   const props = vm.proxy!.$props as VirtualizedProps
 
   return computed(() => {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const _getItemStyleCache = (_: any, __: any, ___: any) => ({})
+    const _getItemStyleCache = (
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      _: any,
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      __: any,
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      ___: any
+    ): Record<string, T> => ({})
```

Comment:
It seems that there is no need to add so many comments here, using `as` should be fine.
```ts
const _getItemStyleCache = (_: any, __: any, ___: any) =>
 ({} as Record<string, T>)
```

Reply:
That makes sense.🤔